tests/test_all.py

```diff
@@ -18,6 +18,14 @@ async def test_api_30():
     assert report.current.date_time is not None
     assert report.hourly_forecast[0].condition.id is not None
     assert report.daily_forecast[0].condition.id is not None
+
+    report = await client.get_air_pollution(LATITUDE, LONGITUDE)
```

Comment:
tests are identical, it's better to extract it to separated method and remove duplicates. also by removing `current` and `forecast` only single test will remain